doc/cs_registers.rst

```diff
@@ -282,6 +288,87 @@ Reset Value: ``0x0000_0000``
+| 20    | R    | **hit:** 0 = Hit indication not supported.                       |
++-------+------+------------------------------------------------------------------+
+| 19    | R    | **select:** 0 = Only address matching is supported.              |
++-------+------+------------------------------------------------------------------+
+| 18    | R    | **timing:** 0 = Break before the instruction at the specified    |
+|       |      | address.                                                         |
++-------+------+------------------------------------------------------------------+
+| 17:16 | R    | **sizelo:** 0 = Match accesses of any size.                      |
++-------+------+------------------------------------------------------------------+
+| 15:12 | R    | **action:** 1 = Enter debug mode on match.                       |
++-------+------+------------------------------------------------------------------+
+| 11    | R    | **chain:** 0 = Chaining not supported.                           |
++-------+------+------------------------------------------------------------------+
+| 10:7  | R    | **match:** 0 = Match the whole address.                          |
++-------+------+------------------------------------------------------------------+
+| 6     | R    | **m:** 1 = Match in M-Mode.                                      |
++-------+------+------------------------------------------------------------------+
+| 5     | R    | zero.                                                            |
++-------+------+------------------------------------------------------------------+
+| 4     | R    | **s:** 0 = S-Mode not supported.                                 |
++-------+------+------------------------------------------------------------------+
+| 3     | R    | **u:** 1 = Match in U-Mode.                                      |
++-------+------+------------------------------------------------------------------+
+| 2     | RW   | **execute:** Enable matching on instruction address.             |
++-------+------+------------------------------------------------------------------+
+| 1     | R    | **store:** 0 = Store address / data matching not supported.      |
++-------+------+------------------------------------------------------------------+
+| 0     | R    | **load:** 0 = Load address / data matching not supported.        |
++-------+------+------------------------------------------------------------------+
+
+Details of these configuration bits can be found in the RISC-V Debug Specification, version 0.13.2 (see Trigger Registers, Section 5.2).
+
+.. _csr-tdata2:
+
+Trigger Data Register 2 (tdata2)
+--------------------------------
+
+CSR Address: ``0x7A2``
+
+Reset Value: ``0x0000_0000``
+
+Accessible in Debug Mode only, and only when trigger support is enabled (using the DbgTriggerEn parameter).
+
+This register stores the address to match against for a breakpoint trigger.
```

Comment:
To clarify, I would change this to "stores the instruction address to match against".